submariner-operator/submariner-metrics-redesign.md

```diff
@@ -124,12 +141,14 @@ spec:
 
 ### Backward Compatibility
 
-Could modified gateway and globalnet metrics services be an issue? Upgrades?
-
 `subctl diagnose` checks for metrics port in firewall. This test will need to be modified to not check for port itself
 but try acessing the metrics irrespective of ports being open or not. This makes this change not backward compatible with
 older versions of `subctl`.
 
+Older versions of `subctl diagnose` will not work these changes. So users will have to make sure they're using compatible
```

Comment:
This is fine IMHO.

Maybe we can add a check in `diagnose` that alerts users using older cli on newer server resources, but we'll have to see how to achieve it.
In any case, it doesn't affect this EP, it's just a thought.

Reply:
I left it here as something to be release noted and/or documented.